data/Sweedman_2006/metadata.yml

```diff
@@ -0,0 +1,602 @@
+    (Western Australia) since 1962.
+  basis_of_record: lab
+  life_stage: adult
+  sampling_strategy: Records of seed germination times have been kept at Kings Park
+    and Botanic Garden since 1962 after the Gardens initiated a Programme to propagate
+    native species required for living collections. The first two decades of germination
+    records were published in booklet form in 1980 and proved to be a very popular
+    source of information for nurseries and enthusiasts alike. The booklet provided
+    practical information about the expected germination time for more than 2000 species
+    of Australian native plants. ^p The records is this appendix have been compiled
+    from more than 40 years of data collection. This information should enable growers
+    and enthusiasts to plan their propagation and nursery operations and not waste
+    time and space waiting for seeds that will not germinate. The data provided in
+    this appendix relate to the length of time for shortest, longest and mean time
+    to germinate for over 2200 species of Australian native plants, together with
+    the number of samples for which germination was recorded. If there was no germination
+    then the sample has been excluded.
+  original_file: none, data extracted from book
+  notes: data extracted from book
+locations: .na
+contexts:
+- context_property: seed pre-germination treatment
+  category: treatment
+  var_in: recommended pretreatment
+  values:
+  - find: .na.character
+    value: unknown
+    description: unknown
+  - find: HW
+    value: hot water
+    description: Soak in very hot water (c. 95°C) for about 2 minutes.
+  - find: SMK
+    value: smoke
+    description: Soak in smoke water at 1:10, 1:100 or 1:1000 dilution for 24 hours.
+  - find: None
+    value: none
+    description: No pre-treatment required.
+  - find: Light
+    value: light
+    description: Sow on surface. Seed need light to germinate.
+  - find: 100°C/3h
+    value: 100°C for 3h
+    description: Heated in oven at 100°C for 3 hours.
+  - find: RB
+    value: remove bracts
+    description: Remove the bracts from around the seed before sowing.
+  - find: 100°C/10m
+    value: 100°C for 10m
+    description: Heated in oven at 100°C for 10 minutes.
+  - find: SMK+GA
+    value: smoke + Gibberellic acid
+    description: Soak in smoke water at 1:10, 1:100 or 1:1000 dilution for 24 hours
+      and soak in 0.1 o/o solution of Gibberellic acid for 24 hours.
+  - find: GA
+    value: Gibberellic acid
+    description: Soak in 0.1 o/o solution of Gibberellic acid for 24 hours.
+  - find: 80°C/3h + SMK
+    value: 80°C for 3h and smoke
+    description: Heated in oven at 80°C for 3 hours and then soaked in smoke water
+      at 1:10, 1:100 or 1:1000 dilution for 24 hours.
+traits:
+- var_in: mean time to germinate
+  unit_in: d
+  trait_name: seed_germination_time
+  entity_type: population
```

Comment:
Are the seeds usually only sourced from one population?

Reply:
We don't know about collection in this case, but I think it is still appropriate, because the values apply to the "population of seeds" used in the experiment, even if they aren't part of a single geographic population. As in it isn't a species-level measurement.